packages/react-sdk/src/components/Whiteboard/SvgCanvas/SvgCanvas.tsx

```diff
@@ -108,6 +109,71 @@ export const SvgCanvas = forwardRef(function SvgCanvas(
     setContainerDimensions({ width, height });
   }, [width, height, setContainerDimensions]);
 
+  const handleCanvasClick = useCallback(() => {
+    if (svgRef.current) {
+      svgRef.current.focus();
+    }
+  }, []);
```

Comment:
This callback seems to be not needed as soon as there is a tab index on svg canvas. Canvas gets focus and accepts keyboard events.